example/particle_leapfrog/particle_leapfrog.cpp

```diff
@@ -175,9 +175,7 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
 
   Kokkos::deep_copy(pmb->exec_space, ids_this_block, ids_this_block_h);
 
-  ParArrayND<int> new_indices;
-  const auto new_particles_mask =
-      swarm->AddEmptyParticles(num_particles_this_block, new_indices);
+  auto newParticlesContext = swarm->AddEmptyParticles(num_particles_this_block);
```

Comment:
```suggestion
  auto new_particles_context = swarm->AddEmptyParticles(num_particles_this_block);
```
style also below and in other files